src/Command/Bulk/AbstractBulkUsersAssignmentsCommand.php

```diff
@@ -139,7 +139,7 @@ protected function processOperationCollection(BulkOperationCollection $bulkOpera
     /**
      * @throws LogicException
      */
-    protected function ensureConsoleOutput(OutputInterface $output): ConsoleOutputInterface
+    private function ensureConsoleOutput(OutputInterface $output): ConsoleOutputInterface
```

Comment:
Should it really be a private method, since the class itself is abstract?

Reply:
It's not a method meant to be overwritten by inherited classes.

Reply:
Yep, but what I meant is:
a) `execute` method is inherited from `Command` class, and left protected, that's ok - classes extending `AbstractBulkUsersAssignmentsCommand` will either use its own implementation of `execute` method, or will just "call" the parent one;
BUT
b) since `AbstractBulkUsersAssignmentsCommand` is an abstract class - making `ensureConsoleOutput` a private method will also make it a "non-callable" from any inheriting class, and it can't be called from the class itself, since class is 1) abstract; 2) non-static


So, if some class will extend `AbstractBulkUsersAssignmentsCommand` and use smth like `parent::execute` in its own `execute` method -> code will fail

Reply:
So the question itself:
either we should reconsider methods' visibility or completely remove it?

Reply:
Ah ok I see your point. In theory yes, there is a possibility for error, so I reverted it to protected.